packages/frontend/src/redux/slices/createFromImplicit/index.js

```diff
@@ -13,6 +14,9 @@ const createFromImplicitSlice = createSlice({
     reducers: {
         setCreateFromImplicitSuccess: (state, action) => {
             state.createFromImplicitSuccess = action.payload;
+        },
```

Comment:
Not a blocker for this PR, but in general we should avoid putting the entire payload of an action into state, as this is generally discouraged; ideally the payload would contain a property with a name indicating what the value actually is, and we would destructure that value from the payload and set it in state. (Ref: https://redux.js.org/style-guide/style-guide#reducers-should-own-the-state-shape)